data/json/items/gun/223.json

```diff
@@ -45,6 +45,42 @@
     "min_cycle_recoil": 1350,
     "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
   },
+  {
+    "id": "famas",
+    "copy-from": "rifle_auto",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "MAS .223", "str_pl": "MAS .223" },
+    "description": "An assault rifle that was until recently used by French armed forces.  While it is famous for its high fire rate, this is a semi-auto only variation imported to the US in the late eighties. It does come with an integral bipod, though.",
```

Comment:
```suggestion
    "description": "A bullpup assault rifle that was used by French armed forces only until recently.  While the FAMAS was famous for its high fire rate, the MAS .223 is a semi-auto only variation imported to the US in the late eighties.  It retains the integral bipod, though.",
```

Reply:
If it's semi-auto only, it's not an assault rifle. 🤔